awx/ui/src/components/WorkflowOutputNavigation/WorkflowOutputNavigation.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import { useParams, Link } from 'react-router-dom';
+import { t } from '@lingui/macro';
+import {
+  Select,
+  SelectOption,
+  SelectGroup,
+  SelectVariant,
+  Chip,
+} from '@patternfly/react-core';
+import ChipGroup from 'components/ChipGroup';
+
+const JOB_URL_SEGMENT_MAP = {
+  job: 'playbook',
+  project_update: 'project',
+  system_job: 'management',
+  system: 'system_job',
+  inventory_update: 'inventory',
+  workflow_job: 'workflow',
+};
+
+function WorkflowOutputNavigation({ relatedJobs }) {
+  const { id } = useParams();
+  const parent = document.getElementById('jobOuputCard');
```

Comment:
this feels wrong, using getElementById, especially with a hard-coded id string (i.e. this component is dependent on particular internals of a parent component and will break if those change). Can this be done by passing in a ref instead?

Reply:
hadn't thought of the ref.  I'll explore